docroot/profiles/custom/cgov_site/themes/custom/cgov/cgov_common/templates/content/cancer_center/node--cgov-cancer-center--full.html.twig

```diff
@@ -1,82 +1,73 @@
               {{ content.field_parent_institution }}
-              {% endif %}
               {% if content.field_institution_type %}
-              <br />
-              {{ content.field_institution_type }}
-              {% endif %}
+                <br/>{{ content.field_institution_type }}
+              {%- endif -%}
             </div>
-            {% if content.field_org_head_name %}
-            <p>
-            {% endif %}
-              {{ content.field_org_head_name }}
-              {% if content.field_org_head_title %}
-              , {{ content.field_org_head_title }}
-              {% endif %}
-            {% if content.field_org_head_name %}
-            </p>
-            {% endif %}
+            {%- if content.field_org_head_name -%}
+              <p>
+                {{ content.field_org_head_name|escape|striptags('<!-- -->')|trim }}
+                {%- if content.field_org_head_title -%}
+                  , {{ content.field_org_head_title|escape|striptags('<!-- -->')|trim }}
+                {%- endif -%}
+              </p>
+            {%- endif -%}
             {% if content.field_address_1 %}
-            <p>
-            {% endif %}
-              {{ content.field_address_1 }}
-              {% if content.field_address_2 %}
-              <br />
-              {{ content.field_address_2 }}
-              {% endif %}
-              {% if content.field_city %}
-              <br />
-              {{ content.field_city }}
-              {% endif %}
-              {% if content.field_state %}
-              , {{ content.field_state }}
-              {% endif %}
-              {% if content.field_zip_code %}
-              {{ content.field_zip_code }}
-              {% endif %}
-            {% if content.field_address_1 %}
-            </p>
+              <p>
+                {{ content.field_address_1 }}
+                {% if content.field_address_2 %}
+                  <br/>{{ content.field_address_2 }}
+                {% endif %}
+                {%- if content.field_state -%}
+                  <br/>{{ content.field_city|escape|striptags('<!-- -->')|trim }}, {{ content.field_state|escape|striptags('<!-- -->')|trim }}
+                {% else %}
+                  <br/>{{ content.field_city }}
+                {% endif %}
+                {{ content.field_zip_code }}
+              </p>
             {% endif %}
             {% if content.field_phone_label_1 and content.field_phone_number_1 %}
-            <p>
-              {{ content.field_phone_label_1 }}: <a href="tel:{{ content.field_phone_number_1 }}" rel="nofollow">{{ content.field_phone_number_1 }}</a>
-            </p>
+              <p>
+                {{ content.field_phone_label_1 }}: <a href="tel:{{ content.field_phone_number_1 }}" rel="nofollow">{{ content.field_phone_number_1 }}</a>
+              </p>
             {% endif %}
             {% if content.field_phone_label_2 and content.phone_number_2 %}
```

Comment:
@alifr There is one more change that I just noticed needs to be made in addition to the merge. The second field here needs to be updated to content.field_phone_number_2.

Reply:
Good catch. I've updated the file.